README.md

```diff
@@ -14,9 +14,9 @@ Current features include:
 * Discovery of servers on the local network
 * Multiple servers
 * Volume management, via Kodi or external device
-* Shutdown / wake-up management
+* Shutdown / Wake-on-LAN management
```

Comment:
Same remark as in translation. This one should stay as Wake Up (without dash from what i've checked), because Wake-on-LAN is just one mechanism to implement the feature (more are planned).

Reply:
Good thinking. If it is three words that otherwise would be together there is a dash. Otherwise it would be wake up management, they are asleep, or somesuch.